includes/class-sensei-utils.php

```diff
@@ -2282,6 +2282,7 @@ public static function generate_drop_down( $selected_value, $options = array(),
 					'class' => array(),
 					'id'    => array(),
 					'name'  => array(),
+					'style'  => array(),
```

Comment:
This arrow needs alignment, as per the linter issue in CI.

Reply:
Ah, just saw my IDE's linter times out on this file. Fixed in 073407e